### DISCUSSION TITLE
Custom Domain taken by other repo

### DISCUSSION THREAD

**Discussion by suyanthapa** [community/community#140242] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

i already checked all other repo of my account but still i am facing the custom domain taken by other repo . what can be the other solution

![error](https://github.com/user-attachments/assets/2b3d5939-85c6-4d06-acc5-88c631230ee8)

**Comment by InboraStudio** [comment]
dont worry i file a complain report from your side it will be seen for sure

**Comment by suyanthapa** [comment, marked as answer]
I finally got it
